pandas/core/indexes/extension.py

```diff
@@ -221,6 +224,20 @@ def __getitem__(self, key):
     def __iter__(self):
         return self._data.__iter__()
 
+    def __len__(self) -> int:
+        return len(self._data)
+
+    @property
+    def size(self) -> int:
+        # If EA ever requires size, this can become self._data.size
+        return len(self)
+
+    @property
+    def dtype(self):
+        return self._data.dtype
```

Comment:
This is already included in the base Index class (with docstring actually), so I think this is not needed here?